diff '--color=auto' -ru a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt	2025-07-17 10:16:44.037234891 +0300
+++ b/CMakeLists.txt	2025-07-17 10:20:36.144579708 +0300
@@ -51,6 +51,7 @@
 message(STATUS "Checking deps...")
 add_subdirectory(hyprland-share-picker)
 
+find_package(sdbus-c++ REQUIRED)
 find_package(Threads REQUIRED)
 find_package(PkgConfig REQUIRED)
 pkg_check_modules(
@@ -67,15 +68,7 @@
   hyprutils>=0.2.6
   hyprwayland-scanner>=0.4.2)
 
-# check whether we can find sdbus-c++ through pkg-config
-pkg_check_modules(SDBUS IMPORTED_TARGET sdbus-c++>=2.0.0)
-if(NOT SDBUS_FOUND)
-  include_directories("subprojects/sdbus-cpp/include/")
-  add_subdirectory(subprojects/sdbus-cpp EXCLUDE_FROM_ALL)
-  add_library(PkgConfig::SDBUS ALIAS sdbus-c++)
-endif()
-
-# same for hyprland-protocols
+# check whether we can find hyprland-protocols through pkg-config
 pkg_check_modules(HYPRLAND_PROTOS IMPORTED_TARGET hyprland-protocols)
 if(HYPRLAND_PROTOS_FOUND)
   set(HYPRLAND_PROTOCOLS "${HYPRLAND_PROTOS_PREFIX}/share/hyprland-protocols")
@@ -86,7 +79,7 @@
 file(GLOB_RECURSE SRCFILES CONFIGURE_DEPENDS "src/*.cpp")
 add_executable(xdg-desktop-portal-hyprland ${SRCFILES})
 target_link_libraries(
-  xdg-desktop-portal-hyprland PRIVATE rt PkgConfig::SDBUS Threads::Threads
+  xdg-desktop-portal-hyprland PRIVATE rt sdbus-c++ Threads::Threads
                                       PkgConfig::deps)
 
 # protocols
