diff --git a/src/window.cpp b/src/window.cpp
index 05e878f..bb628aa 100644
--- a/src/window.cpp
+++ b/src/window.cpp
@@ -248,8 +248,11 @@ void syspower::on_button_clicked(const std::string& button) {
 		button_text = "Rebooting...";
 	}
 	else if (button == "logout") {
-		command += " terminate-user $USER";
 		button_text = "Logging out...";
+		system("uwsm stop");
+		for (const auto &window : windows)
+			window->close();
+		close();
 	}
 	else if (button == "suspend") {
 		button_text = "Suspending...";
